androidComposePrototype/src/main/java/com/genesys/cloud/messenger/androidcomposeprototype/ui/testbed/TestBedViewModel.kt

```diff
@@ -279,7 +278,10 @@ class TestBedViewModel : ViewModel(), CoroutineScope {
         clearCommand()
         val keyValue = customAttributes.toKeyValuePair()
         if (keyValue.first.isNotEmpty()) {
-            client.customAttributesStore.add(mapOf(keyValue))
+            val addSuccess = client.customAttributesStore.add(mapOf(keyValue))
+            if (addSuccess) {
+                onSocketMessageReceived("Custom attribute added: $keyValue")
+            }
```

Comment:
Just a question: if the addSuccess is false, we don't mind? Maybe it is not interested for MessengerSDK...

Reply:
If addSuccess is false the eventHandler catches it.  If this isn't good practice, I will review and update.